package.json

```diff
@@ -11,6 +11,7 @@
     "versions": "node ./scripts/commands.js versions",
     "upload": "node ./scripts/commands.js upload",
     "update_patches": "node ./scripts/commands.js update_patches",
+    "apply_patches": "node ./scripts/sync.js --run_hooks",
```

Comment:
So, my only problem with this is that sometimes patches don't re-apply correctly. It may be ok right now in brave, but I've seen muon either fail to apply some patches or do things like apply patches a second time when they're at the end of a file. Running `init is less than ideal because it typically triggers a large rebuild, but I would like to find a good way to update patches reliably without a large rebuild. Any thoughts?

Reply:
even if we selectively reset the files first it will still trigger a large rebuild because of the dependency tree so we need some way to update only the files that will actually change. Maybe apply in a temp dir and then check or something?

Reply:
I saw it in muon but I haven't seen it in brave yet.  It seems to me that it should be sufficient to teach sync.js to `git checkout` the modified files before applying them, if it doesn't already.  However, all I wanted to address with this change is the front end: I can't remember `npm run sync -- --run_hooks`, apparently, but I can remember `npm run apply_patches` (and grep turns it up too with the keywords 'apply' and 'patch').

Reply:
well, if you do a git checkout before applying the new patches you'll still have a big rebuild. The problem is that some of the files we patch have a lot of deps and therefore trigger a builds in a large number of targets

Reply:
I'm not necessarily opposed to merging this as-is, but everyone should be aware that it isn't necessarily reliable and you should revert back to `init` if you have issues

Reply:
I suppose we could add the `git checkout` to each file before applying the patch for now and that will at least make this reliable

Reply:
then follow-up later with something more efficient